pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -29,8 +29,9 @@ import (
 )
 
 const (
-	keyAccess = "accessKeyId"
-	keySecret = "secretKey"
+	keyAccess            = "accessKeyId"
+	keySecret            = "secretKey"
+	sourceVMInitialState = "sourceVmInitialState"
```

Comment:
according to API conventions annotations should be prefixed with a label, and prefer teh following format:  `vmimport.v2v.kubevirt.io/source-vm-initial-state`

see https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#label-selector-and-annotation-conventions

Reply:
Thanks!

Reply:
Fixed.